artiq/coredevice/trf372017.py

```diff
@@ -59,8 +59,8 @@ class TRF372017:
     ioff = 0x80  # 8b
     qoff = 0x80  # 8b
     vref_sel = 4  # 0.85 V, 3b
-    tx_div_sel = 1  # div2, 2b
-    lo_div_sel = 3  # div8, 2b
+    tx_div_sel = 0  # div1, 2b
```

Comment:
Why change these?

Reply:
The VCO frequency already needed to be changed to accommodate the new PFD default. As these outputs are already changing, it seemed sensible to consolidate their values. Defaulting to `div1` seems sensible, I'm open to suggestions for other values.

Reply:
Sounds good.